crates/matrix-sdk/src/sliding_sync/mod.rs

```diff
@@ -682,10 +681,9 @@ impl SlidingSync {
         let (request, request_config, position_guard) =
             self.generate_sync_request(&mut LazyTransactionId::new()).await?;
 
-        // The code manipulates `Request` and `Response` from Simplified MSC3575
-        // because it's the future standard. If
-        // `Client::is_simplified_sliding_sync_enabled` is turned off, the
-        // Simplified MSC3575 `Request` must be transformed into a MSC3575 `Request`.
+        // The code manipulates `Request` and `Response` from MSC4186 because it's
+        // the future standard. Let's check if the generated request must be
```

Comment:
preexisting: if you say "future", you need to precise what is "now" by including a date :)

Reply:
Yup